packages/eslint-plugin/tests/rules/ban-ts-comment.test.ts

```diff
@@ -187,6 +187,31 @@ if (false) {
         },
       ],
     },
+    {
+      // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
+      code: '// @ts-ignore         ',
```

Comment:
nit
```suggestion
      code: noFormat`// @ts-ignore         `,
```

Reply:
could you direct me to it? I'm sorry, I couldn't find it.

Reply:
oops - I didn't see that you actually included it in your diff suggestion.  thanks!

fixed inhttps://github.com/typescript-eslint/typescript-eslint/pull/2099/commits/a1a8e0ea95a6afcfae2c98a34e713f23c998cc4d